pallets/ethereum-transactions/src/ethereum_transaction.rs

```diff
@@ -33,9 +41,8 @@ impl EthTransactionType {
     pub fn to_abi(&self) -> Result<EthTransactionDescription, ethabi::Error> {
         match self {
             EthTransactionType::PublishRoot(d) => Ok(d.to_abi()),
-            EthTransactionType::DeregisterValidator(d) => Ok(d.to_abi()),
-            EthTransactionType::SlashValidator(d) => Ok(d.to_abi()),
-            EthTransactionType::ActivateValidator(d) => Ok(d.to_abi()),
+            EthTransactionType::ActivateCollator(d) => Ok(d.to_abi()),
+            EthTransactionType::DeregisterCollator(d) => Ok(d.to_abi()),
```

Comment:
Is "re-enabling slashing" captured anywhere in JIRA? We seem to be removing that functionality here.

Reply:
yes we have a ticket for it.